src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup.js

```diff
@@ -1,8 +1,47 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Tr, Th, Td } from '../../../generic/Table/table'
+import { ButtonSecondary } from '../../../generic/buttons'
+import { PopupSubTh, PopupTable } from './ImageAnnotationModal.styles'
 
-const ImageAnnotationPopup = ({ properties }) => {
-  return <div>{properties.benthicAttributeId}</div>
+const ImageAnnotationPopup = ({ properties, getBenthicAttributeLabel, getGrowthFormLabel }) => {
+  return (
+    <PopupTable aria-labelledby="table-label">
+      <thead>
+        <Tr>
+          <Th>Benthic Attribute</Th>
+          <Th>Growth Form</Th>
+          <Th>Confidence</Th>
+        </Tr>
+      </thead>
+      <tbody>
+        <Tr>
+          <PopupSubTh colSpan={3}>Classifier Guesses</PopupSubTh>
+        </Tr>
+        {JSON.parse(properties.annotations).map((annotation) => (
+          <Tr key={annotation.id}>
+            <Td>{getBenthicAttributeLabel(annotation.benthic_attribute)}</Td>
+            <Td>{getGrowthFormLabel(annotation.growth_form)}</Td>
+            <Td>{annotation.score}</Td>
+          </Tr>
+        ))}
+        <Tr>
+          <PopupSubTh colSpan={3}>Add to existing row</PopupSubTh>
```

Comment:
could extract components for` AnnotationRows` and `SectionHeade`r to make it easier to read

ex:

```
const SectionHeader = ({ title }) => (
    <Tr>
      <PopupSubTh colSpan={3}>{title}</PopupSubTh>
    </Tr>
  );
```

and 

```
 const AnnotationRows = ({ annotations }) => (
    <>
      {annotations.map((annotation) => (
        <Tr key={annotation.id}>
          <Td>{getBenthicAttributeLabel(annotation.benthic_attribute)}</Td>
          ```
           ....